daml-lf/transaction/src/main/scala/com/digitalasset/daml/lf/transaction/ContractStateMachine.scala

```diff
@@ -479,12 +508,12 @@ object ContractStateMachine {
     *        2.1. A create will set the corresponding map entry to KeyActive(cid) if the contract has a key.
     *        2.2. A consuming choice on cid will set the corresponding map entry to KeyInactive
     *             iff we had a KeyActive(cid) entry for the same key before. If not, keys
-    *             will not be modified. Later lookups have an activeness check
+    *             will not be modified.
+    *             Later lookups have an activeness check
```

Comment:
I would try to merge this into Point 1, starting at L505. See https://github.com/digital-asset/daml/commit/cf6fdea0eb73c9017e377b1022993dce5ceeddf2#commitcomment-76159212

Reply:
Ok, good to know!